code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -4092,4 +4092,32 @@
 	taste_mult = 2
 	reagent_state = LIQUID
 	nutriment_factor = 40 //very filling
-	color = "#d169b2"
\ No newline at end of file
+	color = "#d169b2"
+
+//ADDITIONS BELOW THIS LINE MADE ON 02/28/2021
+
+/datum/reagent/drink/soda/kompot
+	name = "Kompot"
+	id = "kompot"
+	description = "A traditional eastern european beverage once used to preserve fruit in the 1980s"
+	taste_description = "refreshuingly sweet and fruity"
+	color = "#ed9415" // rgb: 237, 148, 21
+	adj_drowsy = -1
+	adj_temp = -6
+
+	glass_name = "kompot"
+	glass_desc = "A glass of refreshing kompot."
+	glass_special = list(DRINK_FIZZ)
+
+/datum/reagent/ethanol/kvass
+	name = "Kvass"
+	id = "kvass"
+	description = "A traditional fermented Slavic and Baltic beverage commonly made from rye bread."
+	taste_description = "a warm summer day at babushka's cabin"
+	color = "#b78315" // rgb: 183, 131, 21
+	strength = 95 //It's just soda to Russians
```

Comment:
Kvass is like 1-2% alcohol by volume compared to beer which is 4-6%. Seems like a bit of a meme to make it double the strength of beer.

Reply:
Discussed on Discord, higher is weaker in alcohol code. Thanks SS13.